src/components/Transactions/transactions.css

```diff
@@ -42,4 +42,8 @@
 }
 .amount {
     padding: 2rem 0;
+}
+
```

Comment:
Since your are using Styled components and react-material, it's a good idea to generalise your code and avoid use css, in the future thisll will help you to introduce futures where you will need to change styles with your JS code